### ISSUE TITLE
[UNFINISHED] The game is far from finish but if someone would help it could be continued

### ISSUE THREAD

**Issue by VicFinistere** [Zoeinfp/GoForFish#1]
The game is far from finish but if someone would help it could be continued.

**Comment by oranmehavi** [comment]
I think I have found a bug with the character facing the wrong way when going left or right. I got it fixed by just changing the x values in some of the animations. Do you want me to make a pull request for this?

**Comment by VicFinistere** [comment]
Yes please ! Very nice spotted ! I would love to have a first contributor.

**Comment by oranmehavi** [comment]
Made the pull request  #2

**Comment by VicFinistere** [comment]
I don't close this issue because there is more to be done but thanks a lot again ! You can be proud about being an open source contributor on this C# open source Monogame project. Please let me know if I can help you to return the favor.
